tests/Persisters/FlysystemTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
```

Comment:
Nice thorough test!

Reply:
Have now also added tests for the newly introduced `history` mode 👍